### PR TITLE
[FLOC-1282] Display marketing release version in documentation releases.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1282

### CODE REVIEW

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 1]
```diff
@@ -161,6 +171,7 @@ Preparing For a Release
    In addition, review the link-check step of the documentation builder to ensure that all the errors (the links with "[broken]") are expected.
 
 #. Update the staging documentation.
+   (For a documentation release ``${VERSION}`` be the base release version in this step).
```
Comment (tomprince) [inline docs/gettinginvolved/infrastructure/release-process.rst]: This is horrible. I'm going to address this by having a tool that adjusts things automatically.

**Comment by adamtheturtle** [comment]
I should have noticed this in the other branch, apologies, but there should be a test that `UnparseableVersion` is raised. Add that test and merge.

File: docs/gettinginvolved/infrastructure/release-process.rst (hunk 1 above)
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: Do you mean in this branch or another? If another, link to an issue. If this line is staying, change "be" to "is" or "means".

**Comment by adamtheturtle** [comment]
Also, I added some commits to https://github.com/ClusterHQ/flocker/pull/1092 which aren't here.

File: flocker/docs/_version.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    :ivar str commit_hash: The hash of the current commit, or ``None`` if this
+        is a release.
+    :ivar str dirty: If the tree is dirty, the string '-dirty'.
+        Otherwise, ``None``.
+    """
+
+    @property
+    def release(self):
+        """
+        The version string of the last full relase.
```
Comment (adamtheturtle) [inline flocker/docs/_version.py]: s/relase/release. Again should have spotted in the last review.

File: docs/gettinginvolved/infrastructure/release-process.rst (hunk 1 above)
Comment (tomprince) [inline docs/gettinginvolved/infrastructure/release-process.rst]: I inserted a `should` instead.

**Comment by tomprince** [comment]
> Also, I added some commits to #1092 which aren't here.

I merged against the branch, but I guess you deleted it before I fetched it.

**Pull request merged**